api.go

```diff
@@ -241,6 +245,11 @@ type (
 					NftMetadataGatewayUrl string `json:"nftMetadataGatewayUrl"`
 				} `json:"ipfs"`
 			} `json:"export"`
+			TranscodeFile *struct {
```

Comment:
How was this not here yet? 🤔

Reply:
I think this is only used in streamtester, in other projects we use `livepeer-data`. Maybe we should have less common/shared libraries.